lib/rules/declaration-property-max-values/README.md

```diff
@@ -0,0 +1,52 @@
+# declaration-property-max-values
+
+Limit the number of values for a list of properties within declarations.
+
+## Options
+
+`object`: `{ "unprefixed-property-name": int }`
+
+If a property name is surrounded with `"/"` (e.g. `"/^margin/"`), it is interpreted as a regular expression. This allows, for example, easy targeting of shorthands: `/^margin/` will match `margin`, `margin-top`, `margin-inline`, etc.
+
+Given:
+
+```json
+{
+  "border": 1,
+  "/^margin/": 2,
+},
+```
+
+The following patterns are considered problems:
+
+<!-- prettier-ignore -->
+```css
+a { border: 1px solid blue; }
+```
+
+<!-- prettier-ignore -->
+```css
+a { margin: 1px 2px 3px; }
+```
+
+<!-- prettier-ignore -->
+```css
+a { margin: 1px 2px 3px 4px; }
```

Comment:
[trivial] It may be better if there would be a case matching `/^margin/` like `margin-inline`.

Reply:
Hm, in a previous review comment @jeddy3 mentioned that this is not a great example since it is invalid CSS (ex looking at the [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/CSS/margin-inline)), it looks like that `margin-inline` only takes one or two parameters.

I can circumvent this by setting `margin: 1` in the config - would that get the desired effect you're looking for? Happy to rework the tests if that's the case!

Reply:
Thanks, I understand the background. Keeping it as-is may be suitable. 👍🏼 
(@jeddy3 please left a comment if you have about this point)

Reply:
Let's keep as-is.

Reply:
> I can circumvent this by setting margin: 1 in the config - would that get the desired effect you're looking for? Happy to rework the tests if that's the case!

Let's do that, i.e. swap `border` and `margin` around so we still have two different numbers.

```
{
  "border": 2,
  "/^margin/": 1,
},
```

Then we can add `margin-inline` back in.

Reply:
Ah, totally missed this comment earlier - will do now!